### DISCUSSION TITLE
server request capture

### DISCUSSION THREAD

**Discussion by Dragonidas888** [fedorovvl/tso_client#316] [Q&A]
Is there a way to capture all requests and parameters sent by client.exe to the server?

**Comment by fedorovvl** [comment]
u can use charles proxy or fiddler for that

**Comment by Dragonidas888** [comment]
The current client version uses TLS1.2, so I won't be able to use a proxy (all requests are encoded).

**Reply by internety** [reply]
It doesn't prevent anyone from sniffing unencryptted traffic due to certificates are not pinned. that makes possible for anyone to force his pc trust his own, forged, certificate...

**Comment by fedorovvl** [comment]
charles proxy allows you to capture ssl traffic
![изображение](https://github.com/fedorovvl/tso_client/assets/31754742/d70bb657-a1b3-470c-a6f6-2d33c6e2ab4c)

**Comment by Dragonidas888** [comment]
Thank you very much. Charles does it perfectly.
